pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/definition/ActivityDef.java

```diff
@@ -357,6 +360,10 @@ public Boolean getCanDeleteFirstInstance() {
         return canDeleteFirstInstance;
     }
 
+    public boolean isShowActivityStatus() {
```

Comment:
This reads strangely- `showActivityStatus` is already a true/false statement, prefixing the name with `is` isn't necessary

Reply:
renamed